dhall-docs/src/Dhall/data/index.js

```diff
@@ -0,0 +1,28 @@
+function ready(fn) {
```

Comment:
It might be more direct to define things like this:

```javascript
function fn() {
  if (localStorage.getItem(DARK_MODE_OPT) == DARK_MODE_ACTIVE) {
  …
}

if (document.readyState != 'loading'){
  fn();
} else {
  document.addEventListener('DOMContentLoaded', fn);
}
```